src/Compilers/VisualBasic/Portable/Symbols/Source/SynthesizedFieldSymbol.vb

```diff
@@ -59,6 +59,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Return
             End If
 
+            Dim compilation = Me.DeclaringCompilation
+            If Type.ContainsTupleNames() AndAlso
+                compilation.HasTupleNamesAttributes AndAlso
```

Comment:
tracked by [https://github.com/dotnet/roslyn/issues/13940](https://github.com/dotnet/roslyn/issues/13940)

---
In reply to: [79729589](https://github.com/dotnet/roslyn/pull/13923#discussion_r79729589) [](ancestors = 79729589,79716733,79676084)